NitroxClient/GameLogic/Spawning/Metadata/Processor/CyclopsMetadataProcessor.cs

```diff
@@ -65,6 +65,12 @@ private void SetEngineState(GameObject cyclops, bool isOn)
 
     private void SetEngineMode(GameObject cyclops, CyclopsMotorMode.CyclopsMotorModes mode)
     {
+        CyclopsMotorMode.CyclopsMotorModes oldMode = cyclops.GetComponentInChildren<SubControl>(true).cyclopsMotorMode.cyclopsMotorMode;
```

Comment:
I think SubControl is directly on the cyclops so you should be able to use GetComponent instead, and maybe you can cache SubControl when initializing the class instead